model/multi_doc/multi_doc_joint_model.py

```diff
@@ -0,0 +1,43 @@
+from .base_model import MultiDocSummModel
+from model.base_model import SummModel
+from model.single_doc import TextRankModel
+from typing import Union, List
+
+
+class MultiDocJointModel(MultiDocSummModel):
+
+    def __init__(self, model_backend: SummModel = TextRankModel, **kwargs):
+        super(MultiDocJointModel, self).__init__()
+        model = model_backend(**kwargs)
+        self.model = model
+
+    def summarize(self, corpus: Union[List[str], List[List[str]]]) -> List[str]:
+        self.assert_summ_input_type(corpus, None)
+        joint_corpus = []
+        for instance in corpus:
+            joint_corpus.append([" ".join(instance)])
+
+        summaries = []
+        for joint_multi_doc_instance in joint_corpus:
+            summary = self.model.summarize(joint_multi_doc_instance)
+            summaries.append(summary[0])
+
```

Comment:
L14-23 looks strange, why not make `joint_corpus` a `List[str]` and just call `self.model.summarize()` only once?